environments/test.env

```diff
@@ -13,8 +13,8 @@ LP_API_TOKEN=token key
 LP_API_TOKEN_SECRET=token secret
 LP_WEBHOOK_SECRET=dummy-lp-webhook-secret
 
-STORE_API_URL=https://myapps.developer.ubuntu.com/dev/api
-STORE_DEVPORTAL_URL=https://myapps.developer.ubuntu.com/dev
+STORE_API_URL=http://myapps.developer.ubuntu.com/dev/api
+STORE_DEVPORTAL_URL=http://myapps.developer.ubuntu.com/dev
```

Comment:
For some reason nock mocks were failing (not being properly satisfied in tests) when it was `https`.
It seemed like nock was expecting port :443 in URL but it was not there...

Changing test URLs to http resolved the issue. But it's obviously more like a workaround rather that solution.